LuaCATS/Pseudo.d.lua

```diff
@@ -1,8 +1,30 @@
 --- @meta
 
+--- Below refer to the ingame instance IDs
 --- @alias PlayerId integer
 --- @alias AreaId integer
 --- @alias CityId integer
 --- @alias PlotId integer
 --- @alias TeamId integer
 --- @alias UnitId integer
+
+--- Below refer to the database row IDs
```

Comment:
How to call then string types in code? like `TECH_GUILDS` that's inside `Type` Column of `Technologies`
<img width="331" height="47" alt="image" src="https://github.com/user-attachments/assets/6295399f-1025-4fd5-8f78-dfb4790adee1" />